### PR TITLE
workpackage creation frontend

### PR DESCRIPTION
**Pull request by 0xF013**
https://community.openproject.org/work_packages/17549
- [ ] fix autocomplete noscope
- [x] refactor or remove task dropdown menu controller
- [x] fix refresh for /create url
- [x] fix annoying focus on each element on create mode
- [x] focus to title on create toggle
- [x] new work package header in german
    button_save_all: "Save"
    button_save_and_send_all: "Save and send email"
    button_cancel_all: "Cancel"
- [x] column arrows position bug
- [x] time group is now always empty
- [x] payload API title or name
- [x] type switch during creation

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-create-work-package-frontend.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-create-work-package-frontend.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/app/work_packages/directives/inplace_editor/inplace-editor-edit-pane-directive.js [hunk 1]
```diff
@@ -182,13 +185,15 @@ module.exports = function(
...
-      element.bind('keydown keypress', function(e) {
-        if (e.keyCode == 27) {
-          scope.$apply(function() {
-            scope.editPaneController.discardEditing();
-          });
-        }
-      });
+      if (!EditableFieldsState.forcedEditState) {
+        element.bind('keydown keypress', function(e) {
+          if (e.keyCode == 27) {
```
Comment (0xF013) [inline frontend/app/work_packages/directives/inplace_editor/inplace-editor-edit-pane-directive.js]: @houndci and I expected to be an officer on USS Enterprise when I grow up

**Comment by floriank** [comment]
I found some small issues so far:
- somehow the assignee get's two empty options for me.
- the details pane spinner is gone when doing form submission, etc.
- switching from Edit to Create causes the pane to disappear temporarily

Nothing that should prevent an initial merge and a presentation I guess.

**Comment by 0xF013** [comment]
@floriank 
- That's not an empty options, rather a styling issue that accompanies the empty option. Might be worth a high prio premium WP
- The spinner's role ATM is taken by each individual field's blocked state. Can be changed though
- I can't pinpoint the reason for the pane behaviour at this moment

**Comment by floriank** [comment]
> somehow the assignee get's two empty options for me.

That one is my bad - I actually have a user with the name 

``` html
<script>alert(0);</script>
```

for first and last name.

**Comment by floriank** [comment]
@0xF013 after the #3162 has been merged, the branch should probably rebased and the autocompletion stuff can be done, right? right? :cold_sweat:

**Comment by 0xF013** [comment]
@floriank in theory

**Comment by floriank** [comment]
rebased by me against the current `release/4.3`

**Comment by floriank** [comment]
Closed in favour of #3204 (reopenend against the release branch)

**Pull request closed without merging**